python/fusion_engine_client/messages/configuration.py

```diff
@@ -560,6 +568,24 @@ class HardwareTickConfig(NamedTuple):
         "wheel_ticks_to_m" / Float32l,
     )
 
+    class AtmosphericDelayModelConfig(NamedTuple):
+        """!
+        @brief Atmospheric delay model configuration settings.
+
+        See @ref VehicleTickMeasurement.
```

Comment:
Why would we look at VehicleTickMeasurement?

Reply:
Oops darn copy and paste.